lib/my_home_page.dart

```diff
@@ -0,0 +1,117 @@
+  static const double mediumPadding = 5;
+  static const double smallPadding = 2.5;
+  static const Icon appBarBackIcon = Icon(Icons.arrow_back);
+  static const double largeFont = 20;
+  static const double mediumFont = 15;
+
+  final Movie movie = Movie.marioBrosConstructor();
+
+  @override
+  Widget build(BuildContext context) {
+    LikeButton likeButton = LikeButton(movie);
+    return MaterialApp(
+      home: Scaffold(
+        backgroundColor: backgroundColor,
+        appBar: AppBar(
+          backgroundColor: Colors.blue,
+          toolbarHeight: appBarToolbarHeight,
+          title: Text(movie.title),
+          leading: IconButton(
+            onPressed: () {},
+            icon: appBarBackIcon,
+          ),
+        ),
+        body: SafeArea(
+          child: SingleChildScrollView(
+            child: Column(
+              children: [
+                Image.asset(
+                  movie.backdrop,
+                ),
+                Row(
+                  children: [
+                    Image.asset(
+                      movie.poster,
+                      width: posterWidth,
+                    ),
+                    Expanded(
+                      child: Container(
+                        padding: const EdgeInsets.only(
+                          top: largePadding,
+                          bottom: mediumPadding,
+                          left: largePadding,
+                          right: largePadding,
+                        ),
+                        child: Column(
+                          children: [
+                            Text(
+                              movie.originalTitle,
+                              style: const TextStyle(fontSize: largeFont),
+                            ),
+                            Padding(
+                              padding: const EdgeInsets.only(top: largePadding),
+                              child: Text(movie.overview),
+                            ),
+                          ],
+                        ),
+                      ),
+                    ),
+                  ],
+                ),
+                Row(
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.only(
+                        top: smallPadding,
+                        bottom: smallPadding,
+                        left: smallPadding,
+                        right: mediumPadding,
+                      ),
+                      child: likeButton,
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(largePadding),
+                      child: Text('${movie.voteAverage}'),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(largePadding),
+                      child: Text(
+                          '${movie.releaseDate.day}/${movie.releaseDate.month}/${movie.releaseDate.year}'),
```

Comment:
This could be move it to Movie class, you can create a get that return this kind of value like
`class Movie {
    Movie(...)
   String get movieReleaseDate => '${movie.releaseDate.day}/${movie.releaseDate.month}/${movie.releaseDate.year}'
}
`

Reply:
fixed